alpaca_trade_api/rest.py

```diff
@@ -284,10 +285,11 @@ def submit_order(self,
                      trail_percent: str = None):
         """
         :param symbol: symbol or asset ID
-        :param qty: int
         :param side: buy or sell
         :param type: market, limit, stop, stop_limit or trailing_stop
         :param time_in_force: day, gtc, opg, cls, ioc, fok
+        :param qty: float. Mutually exclusive with "notional".
+        :param notional: float. Mutually exclusive with "qty".
```

Comment:
The mutual exclusivity of `qty` and `notional` is captured in the [Orders docs](https://alpaca.markets/docs/api-documentation/api-v2/orders/), so not sure if we need to mention it here.

Reply:
I would document it. it will help to users avoiding this issue. many users just use example code as their baseline